test/components/beta/avatar_test.rb

```diff
@@ -66,55 +66,51 @@ def test_squared_avatar
   def test_renders_link_wrapper
     render_inline(Primer::Beta::Avatar.new(src: "https://github.com/github.png", alt: "github", href: "#given-href"))
 
-    assert_selector("a.avatar") do |(a)|
+    assert_selector("a") do |(a)|
       assert_equal("#given-href", a["href"])
-      assert_selector("img")
-      refute_selector("img.avatar")
+      assert_selector("img.avatar")
```

Comment:
Is it just me or did this change from `refute_selector` to `assert_selector`, and if so, why? I don't see any HTML changes?